docs/modules/deployment/pages/time-series-storage/timeseries/configuration.adoc

```diff
@@ -6,6 +6,11 @@ Set the following property in `$\{OPENNMS_HOME}/etc/opennms.properties` to confi
 [source, properties]
 org.opennms.timeseries.strategy=integration
 
+NOTE: If using a container, it is preferred to set this as an environment variable in your container config instead of as a property file. +
```

Comment:
Is this a preference? Or is it another option?

They'll most likely also have to twiddle other timeseries properties, so they'll have to fiddle with the property file, anyway (as we talked about, we might change this in the future, but this is at least based on how things are *today*).